doc/man3/EVP_DigestInit.pod

```diff
@@ -339,12 +339,13 @@ EVP_MD_CTX_set_params() can be used with the following OSSL_PARAM keys:
 
 =over 4
 
-=item OSSL_PARAM_DIGEST_KEY_XOFLEN <size_t>
+=item OSSL_PARAM_DIGEST_KEY_XOFLEN <unsigned integer>
 
 Sets the digest length for extendable output functions.
-It is used by the SHAKE algorithm.
+It is used by the SHAKE algorithm and should not exceed what can be given
+using a size_t.
```

Comment:
[man-pages(7)](http://man7.org/linux/man-pages/man7/man-pages.7.html) says something about types being bold, so `B<size_t>`

Reply:
Okay, I wasn't sure what the guideline was.
All changed now.